src/components/application/StatsResults.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+
+export default function({ body }) {
```

Comment:
This component should really be named (currently anonymous). This helps to understand the codebase when using the same name when importing.

For consistency the export should also be separate to the definition (the default export may not always be the same thing);
```
function Component({ ...props }) {};
export default Component;
```

Reply:
Good suggestion, I have updated and now is a named function.